portfolio/src/main/webapp/script.js

```diff
@@ -60,6 +60,13 @@ function loadingMyComments() {
     }
 }
 
+function deleteMyComments() {
+  fetch('/delete-data', {method: 'POST'}).then(response => response.text()).then((worked) => {
+    loadingMyComments();
+    console.log(worked);
```

Comment:
Unless this is needed for some reason, we typically remove console logs before submitting.